tensorflow/python/keras/callbacks.py

```diff
@@ -1168,6 +1168,23 @@ class LearningRateScheduler(Callback):
           (integer, indexed from 0) and returns a new
           learning rate as output (float).
       verbose: int. 0: quiet, 1: update messages.
+  
+  ```python
+  # Firstly, let's create a function which 
```

Comment:
Suggesting remove the first sentence "Firstly... to a learning rate."

also increase the length of each line to limit.

Reply:
Ok, will change line length, but what is the line length limit?

Reply:
80 chars per line. Thanks!